chroma-manager/chroma-manager.conf.template

```diff
@@ -207,4 +207,20 @@ server {
     location /iml-srcmap-reverse {
         proxy_pass http://unix:/var/run/iml-srcmap-reverse.sock;
     }
+
+    location /iml-device-aggregator {
+        if ($ssl_client_verify != SUCCESS) {
+            return 401;
+        }
+
+        proxy_set_header X-SSL-Client-On $ssl_client_verify;
+        proxy_set_header X-SSL-Client-Name $ssl_client_s_dn_cn;
+        proxy_set_header X-SSL-Client-Serial $ssl_client_serial;
+
+        proxy_set_header X-Forwarded-Host $host;
+        proxy_set_header X-Forwarded-Server $host;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header Http-Host      $http_host;
+        proxy_pass http://unix:/var/run/device-aggregator.sock;
```

Comment:
What's listening at the other end of this socket?  Can I see the patch for that?  I tried some searches but nothing turned up.

Reply:
@brianjmurrell https://github.com/intel-hpdd/device-aggregator/pull/3